app/models/ability.rb

```diff
@@ -57,7 +57,7 @@ def create_permissions(user=nil, session=nil)
   end
 
   def custom_permissions(user=nil, session=nil)
-    # playlist_permissions
+    playlist_permissions
```

Comment:
Let's move the additional permission methods into ability logic instead of calling them directly from custom_permissions.  See https://github.com/projecthydra/hydra-head/blob/master/hydra-access-controls/lib/hydra/ability.rb#L9-L11